<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>温度</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="temp">
       <property name="text">
        <string>24.6℃</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>湿度</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="hum">
       <property name="text">
        <string>65.1%</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QChartView" name="temp_view">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>391</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AcadEref</family>
      <pointsize>6</pointsize>
     </font>
    </property>
    <property name="interactive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QChartView" name="hum_view">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>380</y>
      <width>391</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AcadEref</family>
      <pointsize>6</pointsize>
     </font>
    </property>
    <property name="interactive">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="temp_btn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>导出Excel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hum_btn">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>导出Excel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>648</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">PyQt5.QtChart</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="pic.qrc"/>
 </resources>
 <connections/>
</ui>
